fuseops/ops.go

```diff
@@ -356,6 +356,9 @@ type CreateFileOp struct {
 	// later call to ReleaseFileHandle.
 	Handle    HandleID
 	OpContext OpContext
+
+	// The flags from the open(2) call, passed through the kernel.
```

Comment:
Here and below, what do you mean with “passed through the kernel”? Do you mean “passed to the kernel”? If so, please change. If not, please explain :)